From 480184f195a228c4c11c6ec913294293db6a310f Mon Sep 17 00:00:00 2001
From: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
Date: Wed, 15 Sep 2021 18:02:45 +0300
Subject: [PATCH 002/233] [FIO extra] apalis_imx6: fix redefinition of
 PHY_ANEG_TIMEOUT

Now the common imx code includes a default timeout for PHY negotiation
(=20000). Remove Apalis specific one to fix complie-time warnings [1].

[1]
include/configs/apalis_imx6.h:57: warning: "PHY_ANEG_TIMEOUT" redefined
   57 | #define PHY_ANEG_TIMEOUT  15000 /* PHY needs longer aneg time */

Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
---
 include/configs/apalis_imx6.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/include/configs/apalis_imx6.h b/include/configs/apalis_imx6.h
index bbdcab29d8f..d4b8390c9d9 100644
--- a/include/configs/apalis_imx6.h
+++ b/include/configs/apalis_imx6.h
@@ -32,9 +32,6 @@
 #define CONFIG_LBA48
 #endif
 
-/* Network */
-#define PHY_ANEG_TIMEOUT		15000 /* PHY needs longer aneg time */
-
 /* USB Configs */
 /* Host */
 #define CONFIG_USB_MAX_CONTROLLER_COUNT		2
-- 
2.25.1

